packages/kbn-i18n/README.md

```diff
@@ -72,19 +72,17 @@ export default function (kibana) {
 }
 ```
 
-The engine uses a locale resolution process similar to that of the built-in
-Intl APIs to determine which locale data to use based on the `accept-language`
-http header.
+The engine uses a `config/kibana.yml` file for locale resolution process.
+If `i18n.locale` option is defined this will be used for localization
 
 The following are the abstract steps i18n engine goes through to resolve the locale value:
 
 - If there's data for the specified locale (localization file is registered in
   `uiExports.translations`), then that locale will be resolved.
 - If locale data is missing for a leaf locale like `fr-FR`, but there is data
```

Comment:
__issue:__ looks like we removed support for that altogether with `accept-language-parser` or I am missing something?